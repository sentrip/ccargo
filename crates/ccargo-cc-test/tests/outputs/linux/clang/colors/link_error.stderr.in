/usr/bin/ld: /tmp/main-43dc4b.o: in function `func':
main.c:(.text+0x0): multiple definition of `func'; /tmp/warn-cdac20.o:warn.c:(.text+0x0): first defined here
clang: [0;1;31merror: [0m[1mlinker command failed with exit code 1 (use -v to see invocation)[0m
