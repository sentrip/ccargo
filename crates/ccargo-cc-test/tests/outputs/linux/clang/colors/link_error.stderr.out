[0m[1m[31merror .text+0x0[0m[1m: [0m multiple definition of `func'; /tmp/warn-cdac20.o:warn.c:(.text+0x0): first defined here[0m[1m[36m
   --> [0mmain.c in function `func`[0m
